tracing-subscriber/src/fmt/fmt_subscriber.rs

```diff
@@ -280,15 +280,26 @@ where
         }
     }
 
+    /// Enable timestamps for log messages using the [`timer`] field in [`Self`].
+    ///
+    /// [`timer`]: super::time::FormatTime
+    pub fn with_timestamp(
+        self,
+        display_timestamp: bool,
+    ) -> Subscriber<C, N, format::Format<L, T>, W> {
+        Subscriber {
+            fmt_event: self.fmt_event.with_timestamp(display_timestamp),
+            fmt_span: self.fmt_span.with_timestamp(display_timestamp),
+            ..self
+        }
+    }
+
     /// Do not emit timestamps with spans and event.
-    pub fn without_time(self) -> Subscriber<C, N, format::Format<L, ()>, W> {
+    pub fn without_time(self) -> Subscriber<C, N, format::Format<L, T>, W> {
```

Comment:
unfortunately, we can't make this change now, as it is a breaking change.

Reply:
Is there a suggested workaround for this or should the PR just be scrapped?